helm/oncall/templates/_env.tpl

```diff
@@ -95,9 +95,16 @@
 {{- end }}
 
 {{- define "snippet.oncall.telegram.env" -}}
+{{- if .Values.telegramPolling.enabled -}}
+{{- $_ := set .Values.oncall.telegram "enabled" true -}}
```

Comment:
Does this mean that setting the telegram polling flag will "automatically" turn on the telegram enabled value?

Reply:
Yes, exactly.